results/covid_can_usa_summary.csv

```diff
@@ -0,0 +1,3 @@
+iso_code,sample_size,mean_response_ratio,median_response_ratio,standard_dev
+CAN,227,71.22982986986875,51.119745222929936,64.4763272822398
+USA,245,17.822632684266615,16.6130767770112,8.480731640224555
```

Comment:
This `.csv` file should go under the `data/processed` and not in the `results` folder.

Can you please move it to the proper place (pointed above)?

Reply:
I actually think it should be under results as the .csv file is part of the EDA results. Also, it would require two output paths if we want to save the .csv in processed file. 

This is related to the reason why we are generating the .csv file in scrip3, because it is part of the EDA analysis, not part of preprocessing data.

Reply:
Then shouldn't we generate a `.png` as well for this table like we're doing for the plots?

I still think that `.csv` files should live under the data folder.

Reply:
Yes!  Agreed. We are already saving the summary statistic table as a png so no need to save the .csv file anyway. 

So can I suggest we do this:
Neel can remove the part where it saves the summary statistic table as a .png in script 4.
I will add it to script 3. So script 3 will generate and save the table as a .png file in results folder. 

Agree? @nphaterp @fatse

Reply:
@nichowu sounds good to me 👍